src/SDK/Sdk.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK;
+
+use OpenTelemetry\API\Metrics\MeterProviderInterface;
+use OpenTelemetry\API\Trace\TracerProviderInterface;
+use OpenTelemetry\Context\Propagation\TextMapPropagatorInterface;
+use OpenTelemetry\SDK\Common\Environment\Accessor;
+use OpenTelemetry\SDK\Common\Environment\EnvironmentVariablesTrait;
+use OpenTelemetry\SDK\Common\Environment\Variables;
+
+class Sdk
+{
+    use EnvironmentVariablesTrait;
+
+    private TracerProviderInterface $tracerProvider;
+    private MeterProviderInterface $meterProvider;
+    private TextMapPropagatorInterface $propagator;
+
+    public function __construct(
+        TracerProviderInterface $tracerProvider,
+        MeterProviderInterface $meterProvider,
+        TextMapPropagatorInterface $propagator
+    ) {
+        $this->tracerProvider = $tracerProvider;
+        $this->meterProvider = $meterProvider;
+        $this->propagator = $propagator;
+    }
+
+    public static function isDisabled(): bool
+    {
+        return Accessor::getBool(Variables::OTEL_SDK_DISABLED);
```

Comment:
I was initially going to suggest using the functions from `EnvironmentVariablesTrait` but those are actually instance methods not static.

I wonder if that's something that should be refactored, as there is no instance access being performed by that trait.

Reply:
Yeah, that's why I didn't use `EnvironmentVariablesTrait` directly. I've had the same thought as you before, the methods in that trait could be static. Are you interested in working on a PR for it?

Reply:
Sure thing! I could definitely refactor that out 👍

Reply:
@brettmc I opened a PR to refactor this at #850.